src/pages/OtherDashboard/OtherDashboard.vue

```diff
@@ -96,17 +97,69 @@ export default {
           company: "Voyager",
           purchased: false,
           when: "Today",
-          today: true,
-        },
+          today: true
+        }
+      ],
+      stocks: [
```

Comment:
Why are you keeping this, I'd suggest you remove the commented out code (unless you want to keep as a structure reminder.